### ISSUE TITLE
Error "cannot convert from 'Microsoft.EntityFrameworkCore.ModelBuilder' to 'DbModelBuilder'

### ISSUE THREAD

**Issue by ollie10** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#523]
Hello I have an error using the beta version v3.0.2 of the EFCore reverse poco.

I'm using these settings to generate my data context:

```C#
Settings.DatabaseType          = DatabaseType.SqlServer;
Settings.TemplateType          = TemplateType.EfCore2;
Settings.GeneratorType         = GeneratorType.EfCore;
```

And after generating the code I have a context that doesn't compile, it doesn't due to this snippet of code that has been generated

```C#
  protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            base.OnModelCreating(modelBuilder);

            modelBuilder.ApplyConfiguration(new PropertyConfiguration());
            modelBuilder.ApplyConfiguration(new PropertyImageConfiguration());
            modelBuilder.ApplyConfiguration(new PropertyMessageConfiguration());

            OnModelCreatingPartial(modelBuilder);
        }
```

and then also those two methods fail

```C#
partial void OnModelCreatingPartial(DbModelBuilder modelBuilder);
static partial void OnCreateModelPartial(DbModelBuilder modelBuilder, string schema);
```

The errors are (in first place):

Argument 1: cannot convert from 'Microsoft.EntityFrameworkCore.ModelBuilder' to 'DbModelBuilder'

and then obviously:

The type or namespace name 'DbModelBuilder' could not be found (are you missing a using directive or an assembly reference?)

Any clue?

I'm referencing these packages:

```Xml
<PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.6" />
<PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.2.6" />
<PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.6" />
<PackageReference Include="Microsoft.Extensions.Identity.Core" Version="2.2.0" />
```

**Comment by sjh37** [comment]
Hi, Can you try again using the latest code found in [DropBox](https://www.dropbox.com/s/c8or3gfr9qokp8h/ReversePoco.7z?dl=0)
I haven't been updating the version numbers, and I should. The code is updated, but still states 3.0.2
Please let me know how you get on :-)

**Comment by sjh37** [comment]
I'm closing this as it has been fixed. Thanks for testing the new version :-)
Send me an email simon@hicrest.net and I'll send you back a 1 year commercial licence.

**Comment by ollie10** [comment]
Hi Simon, thanks for the reply, the problem seem not solved even with the latest version 3.0.3.
Anyway it happens only if you mark **public static string DbContextClassModifiers** as public partial (line 72).
So without this I fixed the problema but it seems more a workaround than a real fix.
What do you mean for commercial license? will it become a payment plugin soon?
Thanks

**Comment by sjh37** [comment]
That should be fixed now, download the latest code from [DropBox](https://www.dropbox.com/s/c8or3gfr9qokp8h/ReversePoco.7z?dl=0)
